Celeste.Mod.mm/Patches/LightningRenderer.cs

```diff
@@ -0,0 +1,103 @@
+﻿#pragma warning disable CS0626 // Method, operator, or accessor is marked external and has no attributes on it
+
+using System;
+using System.Collections;
+using Celeste.Mod;
+using Microsoft.Xna.Framework;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using MonoMod;
+using MonoMod.Cil;
+using MonoMod.InlineRT;
+using MonoMod.Utils;
+
+namespace Celeste {
+    class patch_LightningRenderer : LightningRenderer {
+
+        [MonoModPublic]
```

Comment:
MonoModPublic makes the class and methods public even in the patched executable, which can break reflection.

Reply:
Resolved in commit [6f4ef7f](https://github.com/EverestAPI/Everest/pull/836/commits/6f4ef7fd93c9b4f858749dc54c82c2e1e52a41e8)